rolling-shutter/medley/configuration/traverse.go

```diff
@@ -38,12 +40,31 @@ func extractTags(field *reflect.StructField) tagOptions {
 	}
 	tags := sliceToSet(strings.Split(strings.TrimSpace(tagVal), ","))
 	_, required := tags["required"]
+	_, sensitive := tags["sensitive"]
 	return tagOptions{
 		setExplicitly: true,
 		required:      required,
+		sensitive:     sensitive,
 	}
 }
 
+// GetSensitiveRecursive will return all sensitive config paths.
+func GetSensitiveRecursive(root Config) map[string]any {
```

Comment:
We use `struct{}` as the value type for maps used like sets.  I think we should also do this here as it makes the code a bit clearer. In case I'm  wrong and we need the value, I don't understand why this isn't a bool.